src/components/NewsletterForm/NewsletterFormFooter.tsx

```diff
@@ -5,12 +5,19 @@ import NewsletterFormItem from './NewsletterFormItem';
 export const NewsletterFormFooter = () => (
   <>
     <p className="newsletter-form__item newsletter-form__item--footer">
-      By clicking subscribe, you agree to recieve this newsletter.
+      By clicking subscribe, you agree to receive this newsletter.
     </p>
     <p className="newsletter-form__item newsletter-form__item--footer">
       We use a third party provider, Dotmailer, to deliver our newsletters. For
-      information about how we handle your data, please read our privacy notice.
-      You can unsubscribe at any time using the links in the email you recieve.
+      information about how we handle your data, please read our{' '}
```

Comment:
What's with the `{' '}`?

Is that instead of `&nbsp;`?

Reply:
The code sniffer suggested it  .. on retrospect, figured it's quite a good way of explicitly showing we want a space .. `&nbsp;` isn't technically the right solution I guess, as we actually only require a regular one?